### PR TITLE
Add option for skipping database

### PR DESCRIPTION
**Pull request by ryscher**
On secundus, the database is a read-only replica of production. So we don't want to import new database content (attempting an import will result in an error). 

This change adds the optional argument "skipdb", which will cause the script to skip updates of the database, while still updating the SOLR index.

To test, run the script with and without the "skipdb" parameter.

(Also includes a change of field separator, which was made on secundus but never added to github)

### CODE REVIEW

File: aws-tools/import_recent_dryad.sh [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
 
+# Usage: import_recent_dryad.sh [skipdb]
+#
+# If the 'skipdb' parameter is present, the database is not updated
 
 echo "grabbing latest from aws..."
-/home/ubuntu/.local/bin/aws s3 cp s3://dryad-backup/databaseBackups/dryadDBlatest.sql $HOME/dryadDBlatest.sql 
-import_pg_dump.sh $HOME/dryadDBlatest.sql
 
+if [ "a$1" != "askipdb" ]
```
Comment (daisieh) [inline aws-tools/import_recent_dryad.sh]: What is the reasoning for phrasing the conditional this way, with the prepended "a"?

Comment (ryscher) [reply aws-tools/import_recent_dryad.sh]: It's an old habit for ensuring that you don't get an error if the variable isn't defined. Looking at some examples, this probably is not necessary in bash.

Comment (daisieh) [reply aws-tools/import_recent_dryad.sh]: Can you change it?

Comment (ryscher) [reply aws-tools/import_recent_dryad.sh]: done

**Comment by daisieh** [comment]
Looks fine now, works fine.

**Pull request merged**